src/Form/EntityLimitAddForm.php

```diff
@@ -121,8 +121,10 @@ public function form(array $form, FormStateInterface $form_state) {
           '#title' => $this->t('Select @entity_type bundles', array('@entity_type' => $entity_types[$entity_type])),
           '#description' => $this->t('Select bundles to apply limit.'),
           '#options' => $options,
-          '#default_value' => $entity_limit->getEntityLimitBundles(),
         );
+        if($default_value = $entity_limit->getEntityLimitBundles()){
```

Comment:
space

Reply:
infact use ternary operator (inline if)

Reply:
Changes Done, Included directly in 8.x-2.x branch using the command line.